### PR TITLE
John b issue51

### PR DESCRIPTION
**Pull request by bakerjoh000**

### CODE REVIEW

File: app/templates/search.hbs [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+<div id="bg">
+  <img src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/welcomeBG3.jpg"/>
+</div>
+<div class="wrapper">
+<img class="search-page-logo" src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/waldo_logoColor.png"/>
+
+<h3 class="search-page-slogan"> Let's find the job you've been looking for. </h3>
+
+<div class="div-search-bar">
+{{search-bar
```
Comment (ekraihan) [inline app/templates/search.hbs:11]: Maybe for the search bar and the up slider you could use the real images? I think the search logo is the white search logo and you can put it ontop of a red div

File: app/templates/search.hbs [hunk 2]
```diff
@@ -1,2 +1,40 @@
-Search Page
-{{outlet}}
+
+<div id="bg">
+  <img src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/welcomeBG3.jpg"/>
+</div>
+<div class="wrapper">
+<img class="search-page-logo" src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/waldo_logoColor.png"/>
```
Comment (ekraihan) [inline app/templates/search.hbs]: BTW, I made a helper for icons and images. This image could be defined by 
`src={{image-asset "waldo_logoColor.png"}}`

the background could be `src={{icon-asset "welcomeBG3.jpg"}}`

just so ya know :)

**Review by ekraihan — APPROVED** [review]

File: app/templates/search.hbs [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+  image="https://png.icons8.com/search-filled/ios7/39/00B5AF"
+}}
+</div>
+<div id="recommended-jobs-header">
+   <h3 id="recommended-jobs-title"> Our Top Picks for You. </h3>
+    <div {{action "showRecommendedJobSlider"}}><img src="https://png.icons8.com/back-to/color/50/e74c3c"></div>
+    <div id="recommended-jobs-body" class="show">
+        {{#each model as |user|}}
+
+        <div class="list">
```
Comment (ekraihan) [inline app/templates/search.hbs:22]: I believe Jeremy is making a component for a job search listing, so at some point, we can come back and make this use his component 👍

File: app/templates/search.hbs (hunk 2 above)
Comment (bakerjoh000) [reply app/templates/search.hbs]: Nice,  will use this.

**Pull request merged**